phaseshifts/gui/elements_gui.py

```
@@ -54,10 +54,10 @@
 import wxversion
 
 wxversion.ensureMinimal("2.8")
-import wx
-from wx.lib import fancytext, buttons, rcsizer
+import wx  # noqa: E402
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)